test/should-work-with-tsx/foo.tsx

```diff
@@ -0,0 +1,13 @@
+function MainButton () {}
+
+module.exports = function TodoList () {
+  return (
+    <>
+      <MainButton
+        src='https://i.imgur.com/yXOvdOSs.jpg'
+        alt='Hedy Lamarr'
+        className='photo'
+      />
+    </>
+  )
+}
```

Comment:
I'm not familiar with the validation against external repos: do they include repos with JSX/TSX code?

Reply:
There are some currently disabled one: https://github.com/neostandard/neostandard/blob/885e1985bc9a84a1fc5441a4894fc4036ded8a6f/dependents-data/known-failures/standard.json#L13

Should reactivate them

Reply:
Reactivated all that could possibly be including JSX, and added reasons for the deactivating the ones that still didn't pass